lib/build/runtimeos.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2022 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package build
+
+//go:generate /bin/sh runtimeos.sh
```

Comment:
Not very portable.

Reply:
Does it need to be though? This requires regeneration Literally Never(tm) :)

Reply:
Wouldn't this trip whenever someone just runs go generate in the project?

Reply:
If someone did, yes, I guess it would. But then they also need to have all the protobuf tools, the mock generator and whatnot installed. I don't think it's a very common thing for anyone to do on a lark. I mean, I could just remove the generator, the generated constants themselves are trivial and could be hand maintained after the initial generation too...

Reply:
There, generator is gone; shell script remains for provenance or in case we ever want to update.